src/bosh-director/lib/bosh/director/jobs/update_deployment.rb

```diff
@@ -35,15 +35,15 @@ def perform
           @event_log.warn_deprecated(warning)
           cloud_config_models = nil
         else
-          cloud_config_models = Bosh::Director::Models::Config.find_by_ids(@cloud_config_ids)
+          cloud_config_models = Bosh::Director::Models::Config.where(id: @cloud_config_ids).to_a
```

Comment:
Is there a reason to change this from using `find_by_ids`? From the definition on the class it seems to be doing a very similar thing.

Reply:
I thought it should be working as is, but the tests showed it was not actually working.  Without this change, I was unable to get the `cloud_config_ids` to result in cloud_config objects passed to the test doubles.

Reply:
I see the issue now - `find_by_ids` is actually a custom find method defined on our model (I had assumed it was provided by Sequel).  I'll investigate a bit further to see why it's not finding the models we're creating for test.

Reply:
We were stubbing its return value in the test.  I removed the stub and restored the original code.  Now this PR is just adding some new test coverage.